src/design_docs/org/lib/app.coffee

```diff
@@ -1,22 +1,38 @@
 _ = require('./underscore')
 h = require('./helpers')
-validation = require('./validation/validate')
+validate = require('./validation/validate')
 actions = require('./actions')
 audit = require('./shared/audit')
 
-dd =
-  views: {}
+auth = validate.auth
 
+dd =
+  views:
+    by_role:
+      map: (doc) ->
+        validate = require('views/lib/validation/validate')
+        if not validate._is_team(doc)
+          return
+        team_id = doc._id.slice(5)
```

Comment:
my concern with replace is it would replace team_ anywhere in the string, wouldn't it?